LayoutTests/platform/glib/TestExpectations

```diff
@@ -1930,10 +1930,18 @@ imported/w3c/web-platform-tests/svg/animations/reinserting-svg-into-document.htm
 imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-opacity.html [ ImageOnlyFailure ]
 
 # The following tests need LBSE enabled builds, which is off by default for this port.
-svg/compositing [ Skip ]
 svg/z-index [ Skip ]
 svg/foreignObject/respect-block-margin.html [ Skip ]
 
+svg/compositing/outermost-svg-directly-composited-group-child.html  [ ImageOnlyFailure ]
+svg/compositing/outermost-svg-directly-composited-shape-child.html  [ ImageOnlyFailure ]
+svg/compositing/outermost-svg-directly-composited-transformed-group-child.html  [ ImageOnlyFailure ]
+svg/compositing/outermost-svg-with-border-overflow-visible.html  [ ImageOnlyFailure ]
+svg/compositing/outermost-svg-with-border-padding-margin.html  [ ImageOnlyFailure ]
+svg/compositing/outermost-svg-with-border-padding.html  [ ImageOnlyFailure ]
+svg/compositing/outermost-svg-with-border.html  [ ImageOnlyFailure ]
+svg/compositing/svg-poster-circle.html  [ ImageOnlyFailure ]
```

Comment:
Would be great if you could comment the test results, at least in the PR. The commit message also doesn't mention yet that you unskipped tests -- would be natural to explain the changes there.

Reply:
I added a comment regarding these tests.